playbooks/ansible-test-base/post.yaml

```diff
@@ -1,10 +1,20 @@
 ---
 - hosts: controller
   tasks:
+    - name: Get collection namespace
```

Comment:
for this, we don't want to modify ansible-test-base/post.yaml, as this is limited to units.

Lets, create ansible-test-units-base/post.yaml not to affect other jobs.